### DISCUSSION TITLE
Have Europe API usage limits changed?

### DISCUSSION THREAD

**Discussion by andyearnshaw** [Hyundai-Kia-Connect/kia_uvo#836] [Q&A]
I couldn't start warming my car up this morning, I got the following message in the Bluelink app:

> The maximum number of daily vehicle checks has been exceeded. Please be aware that using 3rd party apps may contribute to exceeding this limit.

When I checked Home Assistant, it appears everything went to _Unavailable_ at 4am. I have the following configuration:

<img width="598" alt="image" src="https://github.com/Hyundai-Kia-Connect/kia_uvo/assets/1826067/5307adc2-9f2f-4afb-8167-6df2588740dd">

My force refresh blackout is set to 10pm-7am, so there should have been no force refreshes. I only have three automations that interact with Bluelink and none of them ran between midnight and 4am, only two were triggered yesterday. I haven't changed any behaviours or patterns, I mostly use the app and more or less only use the integration for automations.

The API limits suggest 200 requests per day. It seems unlikely I'd hit that before 4am, especially if I haven't changed anything. Has anyone else experienced something similar?

**Comment by cdnninja** [comment]
Rumour has it they are lowering it.   

Crank up that force update interval. It can impact your 12v battery.   Try set that to 1440.
